src/server.ts

```diff
@@ -19,6 +19,7 @@ const initializeServer = () => {
   server.patch("/projects/:projectId/columns/:columnId", column.editColumnById);
 
   // ticket
+  server.get("/projects/:projectId/tickets/:ticketId", ticket.getTicketById);
```

Comment:
we can reconsider how the endpoint should be.
I think it is better not to have columnId here.